Cargo.toml

```diff
@@ -21,7 +21,7 @@ members = ["xtask"]
 
 [dependencies]
 parking_lot_core = { version = "0.9.10", optional = true, default-features = false }
-portable-atomic = { version = "1.7", optional = true, default-features = false }
+portable-atomic = { version = "1.8", optional = true, default-features = false }
```

Comment:
Let’s maybe keep it at 1.7? The users are going to get 1.8 or later by default anyway, but it’s nice to keep an option for users with older compilers to use the older version.

(note: as this is an optional dep, it is out of scope for the declared MSRV policy, so  not actually against bumping msrv with portable atomic on. But if we can avoid that, that would be a kind thing to do for some subset of the users)

Reply:
portable-atomic's MSRV has always been Rust 1.34, whether 1.7, 1.8 or the current latest version (1.9).

The use of new compiler features to improve diagnostics has been done in a way that does not break compatibility with older compilers.